contrib/storage-hive/core/src/main/java/org/apache/drill/exec/store/hive/HiveAbstractReader.java

```diff
@@ -218,17 +229,18 @@ private void init() throws ExecutionSetupException {
         }
       }
     } catch (Exception e) {
-      throw new ExecutionSetupException("Failure while initializing HiveRecordReader: " + e.getMessage(), e);
+      throw new ExecutionSetupException("Failure while initializing HiveOrcReader: " + e.getMessage(), e);
```

Comment:
"HiveOrcReader:" in the exception-string should be changed so that it prints the actual reader type.